src/Symfony/Component/Form/FormInterface.php

```diff
@@ -17,6 +17,10 @@
  * A form group bundling multiple forms in a hierarchical structure.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @template T
+ *
+ * @method void clearErrors(bool $deep)
```

Comment:
why this?
this should be documented in changelog+upgrade if we really want to add this method to the interface

Reply:
I have absolutely no clue. This was in the psalm stubs and seemed odd to me, but then again I never touched the Forms component. I will remove it.